### ISSUE TITLE
ipn/ipnlocal: spurious "[unexpected] peerapi listen" log lines

### ISSUE THREAD

**Issue by josharian** [tailscale/tailscale#2395]
Our corp e2e test always includes unexpected lines like this:

```
2021-07-12T17:09:56.5924798Z     logger.go:31: n2: [unexpected] peerapi listen("100.98.201.112") error: listen tcp4 100.98.201.112:0: bind: cannot assign requested address
```

They don't seem to be doing any harm. We should either remove the log line, remove the "[unexpected]", or find a way to better distinguish problematic cases from routine failures.

cc @bradfitz

**Comment by wevsmy** [comment]
I've had this problem before

```log
2021/09/01 10:14:20 [unexpected] peerapi listen("fd7a:115c:a1e0:efe3:f632:33af:aaac:21f") error: listen tcp6 [fd7a:115c:a1e0:efe3:f632:33af:aaac:21f]:0: bind: cannot assign requested address
```

**Comment by DentonGentry** [comment]
I suspect this will be resolved in version 1.22, where peerapi will use netstack.

**Comment by DentonGentry** [comment]
Believed fixed in Tailscale 1.22 and later.
